.github/workflows/build-test.yml

```diff
@@ -99,16 +98,15 @@ jobs:
 
  # This job builds & runs iceoryx tests in Windows 2019
   iceoryx-windows:
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
Once we have updated and aligned the quality levels in `CONTRIBUTING.md` with the ROS ones, we should document it there.

Reply:
Same for MacOS of course

Reply:
i added it as a point to #425